sdk/go/common/workspace/loaders.go

```diff
@@ -145,7 +146,7 @@ func LoadProjectBytes(b []byte, path string, marshaller encoding.Marshaler) (*Pr
 }
 
 // LoadProjectStack reads a stack definition from a file.
-func LoadProjectStack(project *Project, path string) (*ProjectStack, error) {
+func LoadProjectStack(project *Project, path string, diags diag.Sink) (*ProjectStack, error) {
```

Comment:
```suggestion
func LoadProjectStack(diags diag.Sink, project *Project, path string) (*ProjectStack, error) {
```
Consistency, sink first

Reply:
Should be all updated now. In the case where there was a `context.Context`, I left that as the first argument and put the sink as 2nd.